### PR TITLE
Layout Changes and Bug Fix

### PR DESCRIPTION
**Pull request by lilyclements**

### CODE REVIEW

File: instat/dlgInsertColumn.vb [hunk 1]
```diff
@@ -259,7 +248,11 @@ Public Class dlgInsertColumn
             ucrReceiverColumnsToInsert.Visible = True
             ucrSelectorInseertColumns.Visible = True
         End If
-
+        If rdoBeforeAfter.Checked Then
+            ucrInputBeforeAfter.Enabled = True
+        Else
+            ucrInputBeforeAfter.Enabled = False
```
Comment (dannyparsons) [inline instat/dlgInsertColumn.vb]: If this is needed, we should also be clearing the receiver?

Comment (lilyclements) [inline instat/dlgInsertColumn.vb]: Oh yes of course, I've added that in now

Comment (dannyparsons) [inline instat/dlgInsertColumn.vb]: Sorry I got that wrong, I thought it was a receiver. I think this should just be part of checks above to make this visible/invisible when not on that option. Or is this option special?

Comment (dannyparsons) [inline instat/dlgInsertColumn.vb]: Ok yes I remember this now, thanks.

**Pull request merged**